ts/Series/FlowMap/FlowMapSeries.ts

```diff
@@ -0,0 +1,834 @@
+/* *
+ *
+ *  (c) 2010-2022 Askel Eirik Johansson, Piotr Madej
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *  Imports
+ *
+ * */
+
+import type FlowMapSeriesOptions from './FlowMapSeriesOptions';
+import type { MarkerEndOptions } from './FlowMapPointOptions';
+import type SVGPath from '../../Core/Renderer/SVG/SVGPath';
+import FlowMapPoint from './FlowMapPoint.js';
+import MapSeries from '../Map/MapSeries.js';
+import SeriesRegistry from '../../Core/Series/SeriesRegistry.js';
+
+const {
+    series: {
+        prototype: {
+            pointClass: Point
+        }
+    },
+    seriesTypes: {
+        column: ColumnSeries,
+        mapline: MapLineSeries
+    }
+} = SeriesRegistry;
+import { StatesOptionsKey } from '../../Core/Series/StatesOptions';
+import SVGAttributes from '../../Core/Renderer/SVG/SVGAttributes';
+import U from '../../Core/Utilities.js';
+const {
+    defined,
+    extend,
+    merge,
+    pick,
+    addEvent,
+    arrayMax,
+    arrayMin
+} = U;
+
+/* *
+ *
+ *  Declarations
+ *
+ * */
+
+declare module '../../Core/Series/SeriesOptions' {
+    interface SeriesStateHoverOptions {
+
+    }
+}
+
+/**
+ * The flowmap series type
+ *
+ * @private
+ * @class
+ * @name Highcharts.seriesTypes.flowmap
+ *
+ * @augments Highcharts.Series
+ */
+class FlowMapSeries extends MapLineSeries {
+
+    /* *
+     *
+     *  Static properties
+     *
+     * */
+
+    /**
+     * A flowmap series is a series laid out on top of a map series allowing to
+     * display route paths (e.g. flights or ships routes) or flows on a map. It
+     * is drawing a link between two mappoints.
```

Comment:
Maybe a better description would be _"It creates a link between two mappoints."_ ?